flow/scripts/def2odb.tcl

```diff
@@ -0,0 +1,15 @@
+source $::env(SCRIPTS_DIR)/read_liberty.tcl
+
+read_lef $::env(TECH_LEF)
+read_lef $::env(SC_LEF)
+if {[info exist ::env(ADDITIONAL_LEFS)]} {
+  foreach lef $::env(ADDITIONAL_LEFS) {
+    read_lef $lef
+  }
+}
+
+if {[info exist ::env(DEF_FILE)]} {
+  read_def $::env(RESULTS_DIR)/$::env(DEF_FILE)
+  write_db $::env(RESULTS_DIR)/$::env(ODB_FILE)
+}
```

Comment:
I think the `if` statement is not needed here, when the variable does not exist the script would fail silently.